build.gradle

```diff
@@ -208,23 +214,19 @@ task runPhonenumberHugo (dependsOn: 'classes', type: JavaExec) {
 
 task runferHighlow (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.highestlowest.fernanda.Main'
-    standardInput=System.in
```

Comment:
Hi @sergioale9 , thanks for solving the conflicts, please restore the lines like this so tasks do not get broken

Reply:
Sorry Jose, I already repair the file with the lines deleted